README.md

```diff
@@ -170,6 +170,14 @@ You can pass any mocha argv.
 - `-x` add dir ignore coverage, support multiple argv
 - `--prerequire` prerequire files for coverage instrument, you can use this options if load files slowly when call `mm.app` or `mm.cluster`
 - `--typescript` / `--ts` enable typescript support, default to `false`, if true, will auto add `.ts` extension and ignore `typings` and `d.ts`.
+- `--nyc` nyc instruments passthrough. you can use this to overwrite egg-bin's default nyc instruments and add additional ones.
+  >if you want to add addtional nyc reporters, you need to use this rather than add `reporter` key in `.nycrc` because:
```

Comment:
markdown style, if you want to use `ref`, should add space after `>`

or change to `  - if` for secondary list